integrationtest/wildfly/pom.xml

```diff
@@ -374,6 +374,39 @@
                             </resources>
                         </configuration>
                     </execution>
+                    <!-- Temporary workaround for HHH-11394 -->
```

Comment:
I don't mind either solution. I just thought you may have overlooked this potential issue with `git bisect`, but obviously you haven't. So let's keep it that way :)